src/App.tsx

```diff
@@ -1,173 +1,42 @@
-          </a>
+import { Navigate, Route, Routes } from 'react-router-dom';
+import { HomePage } from './components/HomePage/HomePage';
+import { PageNotFound } from './components/PageNotFound/PageNotFound';
+import { PageNavLink } from './components/PageNavLink/PageNavLink';
+import { PeoplePage } from './components/PeoplePage/PeoplePage';
+
+export const App = () => {
+  return (
+    <div data-cy="app">
+      <nav
+        data-cy="nav"
+        className="navbar is-fixed-top has-shadow"
+        role="navigation"
+        aria-label="main navigation"
+      >
+        <div className="container">
+          <div className="navbar-brand">
+            <PageNavLink to="/" text="Home" />
+            <PageNavLink to="/people" text="People" />
+          </div>
         </div>
-      </div>
-    </nav>
-
-    <main className="section">
-      <div className="container">
-        <h1 className="title">Home Page</h1>
-        <h1 className="title">People Page</h1>
-        <h1 className="title">Page not found</h1>
-
-        <div className="block">
-          <div className="box table-container">
-            <Loader />
-
-            <p data-cy="peopleLoadingError" className="has-text-danger">
-              Something went wrong
-            </p>
-
-            <p data-cy="noPeopleMessage">
-              There are no people on the server
-            </p>
-
-            <table
-              data-cy="peopleTable"
-              className="table is-striped is-hoverable is-narrow is-fullwidth"
-            >
-              <thead>
-                <tr>
-                  <th>Name</th>
-                  <th>Sex</th>
-                  <th>Born</th>
-                  <th>Died</th>
-                  <th>Mother</th>
-                  <th>Father</th>
-                </tr>
-              </thead>
-
-              <tbody>
-                <tr data-cy="person">
-                  <td>
-                    <a href="#/people/jan-van-brussel-1714">
-                      Jan van Brussel
-                    </a>
-                  </td>
-
-                  <td>m</td>
-                  <td>1714</td>
-                  <td>1748</td>
-                  <td>Joanna van Rooten</td>
-                  <td>Jacobus van Brussel</td>
-                </tr>
+      </nav>
 
-                <tr data-cy="person">
-                  <td>
-                    <a href="#/people/philibert-haverbeke-1907">
-                      Philibert Haverbeke
-                    </a>
-                  </td>
+      <main className="section">
+        <div className="container">
+          <Routes>
+            <Route path="/" element={<HomePage />} />
+            <Route path="/home" element={<Navigate to="/" replace />} />
 
-                  <td>m</td>
-                  <td>1907</td>
-                  <td>1997</td>
+            <Route path="/people">
+              <Route index element={<PeoplePage />} />
+              <Route path=":selectedSlug" element={<PeoplePage />} />
```

Comment:
you can just name it like a `slug`
```suggestion
              <Route path=":slug" element={<PeoplePage />} />
```